src/sentry/static/sentry/app/actions/settingsBreadcrumbActions.jsx

```diff
@@ -0,0 +1,11 @@
+import Reflux from 'reflux';
+
+const SettingsBreadcrumbActions = Reflux.createActions(['mapTitle', 'trimMappings']);
+
+export const normalizeRoutes = routes =>
```

Comment:
This belongs in `app/utils` - can you add a comment describing what it does and a test?